packages/plugin-core/src/commands/pods/GoogleDocsExportPodCommand.ts

```diff
@@ -118,6 +125,31 @@ export class GoogleDocsExportPodCommand extends BaseExportPodCommand<
       return;
     }
 
+    let parentFolderId = opts?.parentFolderId;
```

Comment:
can we store this in the note as well? so if the gdoc was already exported to a folder, we continue exporting it to the same folder

Reply:
when we export a note again, it always [overwrites](https://github.com/dendronhq/dendron/blob/756395f8a50ab8f38b405a9f61a98dc968e9a2f3/packages/pods-core/src/v2/pods/export/GoogleDocsExportPodV2.ts#L276) the existing document.

Reply:
when i say store in the note, i meant the frontmatter of the note in dendorn. like we do with `uri`